libs/platform/src/lib/thumbnail/thumbnail-image/thumbnail-image.component.html

```diff
@@ -6,7 +6,7 @@
         [class.fdp-thumbnail-image--selected]="image.selected"
         [class.fdp-thumbnail-image--overlay]="image.overlayRequired"
         (click)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick(image)"
```

Comment:
you changed `thumbnailClick` signature, from now the first parameter must be event